packages/aggregator/src/aggregationTypes.js

```diff
@@ -32,41 +32,203 @@ const SUM_PREVIOUS_EACH_DAY = 'SUM_PREVIOUS_EACH_DAY';
-  { code: MOST_RECENT, description: 'MOST_RECENT' },
-  { code: OFFSET_PERIOD, description: 'OFFSET_PERIOD' },
-  { code: RAW, description: 'RAW' },
-  { code: REPLACE_ORG_UNIT_WITH_ORG_GROUP, description: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP' },
-  { code: SUM_EACH_QUARTER, description: 'SUM_EACH_QUARTER' },
-  { code: SUM_EACH_YEAR, description: 'SUM_EACH_YEAR' },
-  { code: SUM_EACH_MONTH, description: 'SUM_EACH_MONTH' },
-  { code: SUM_EACH_WEEK, description: 'SUM_EACH_WEEK' },
-  { code: SUM_MOST_RECENT_PER_FACILITY, description: 'SUM_MOST_RECENT_PER_FACILITY' },
-  { code: SUM_PER_ORG_GROUP, description: 'SUM_PER_ORG_GROUP' },
-  { code: SUM_PER_PERIOD_PER_ORG_GROUP, description: 'SUM_PER_PERIOD_PER_ORG_GROUP' },
-  { code: SUM_PREVIOUS_EACH_DAY, description: 'SUM_PREVIOUS_EACH_DAY' },
-  { code: SUM_UNTIL_CURRENT_DAY, description: 'SUM_UNTIL_CURRENT_DAY' },
-  { code: SUM, description: 'SUM' },
+    description:
+      'Get the latest response for each quarter within selected period, fill each quarter with the latest value till current quarter',
+    schema: getSchema(
+      FINAL_EACH_QUARTER_FILL_EMPTY_QUARTERS,
+      DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA,
+    ),
+  },
+  {
+    code: FINAL_EACH_QUARTER,
+    description: 'Get the latest response for each quarter within selected period',
+    schema: getSchema(FINAL_EACH_QUARTER, DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA),
+  },
+  {
+    code: FINAL_EACH_WEEK_FILL_EMPTY_WEEKS,
+    description:
+      'Get the latest response for each week within selected period, fill each week with the latest value till current week',
+    schema: getSchema(FINAL_EACH_WEEK_FILL_EMPTY_WEEKS, DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA),
+  },
+  {
+    code: FINAL_EACH_WEEK,
+    description: 'Get the latest response for each week within selected period',
+    schema: getSchema(FINAL_EACH_WEEK, DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA),
+  },
+  {
+    code: FINAL_EACH_YEAR_FILL_EMPTY_YEARS,
+    description:
+      'Get the latest response for each year within selected period, fill each year with the latest value till current year',
+    schema: getSchema(FINAL_EACH_YEAR_FILL_EMPTY_YEARS, DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA),
+  },
+  {
+    code: FINAL_EACH_YEAR,
+    description: 'Get the latest response for each year within selected period',
+    schema: getSchema(FINAL_EACH_YEAR, DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA),
+  },
+  {
+    code: MOST_RECENT_PER_ORG_GROUP,
+    description: 'Legacy aggregation type, only used in legacy data builder',
+    schema: getSchema(MOST_RECENT_PER_ORG_GROUP),
+  },
+  {
+    code: MOST_RECENT,
+    description: 'Get the latest data within selected period',
+    schema: getSchema(MOST_RECENT, DATA_SOURCE_ENTITY_TYPE_REQUIRED_SCHEMA),
+  },
+  {
+    code: OFFSET_PERIOD,
+    description: 'Modify the period from the data',
+    schema: getSchema(OFFSET_PERIOD, OFFSET_PERIOD, OFFSET_PERIOD_SCHEMA),
```

Comment:
Does getSchema take 3 arguments or just 2?

Reply:
Just 2, thanks for pointing.